integrationtest/tuf/upload_secrets_test.go

```diff
@@ -0,0 +1,154 @@
+/*
+Copyright 2018 Google Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package integrationtest
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"math/rand"
+	"os"
+	"strings"
+	"testing"
+
+	yaml "gopkg.in/yaml.v2"
+
+	"github.com/GoogleCloudPlatform/runtimes-common/tuf/cmd"
+	"github.com/GoogleCloudPlatform/runtimes-common/tuf/config"
+	"github.com/GoogleCloudPlatform/runtimes-common/tuf/gcs"
+	"github.com/GoogleCloudPlatform/runtimes-common/tuf/kms"
+	"github.com/GoogleCloudPlatform/runtimes-common/tuf/testutil"
+)
+
+var rootSecret = fmt.Sprintf("This is root file secret number %d", rand.Int())
+var targetSecret = fmt.Sprintf("This is target file secret number %d", rand.Int())
+var snapshotSecret = fmt.Sprintf("This is snapshot file secret number %d", rand.Int())
+
+var testFiles = make([]string, 0)
+
+func createAndWriteFile(filename string, text string) string {
+	tmpFile, err := ioutil.TempFile("", filename)
+	if err != nil {
+		panic(fmt.Sprintf("Cannot run tests due to %v", err))
+	}
+	testFiles = append(testFiles, tmpFile.Name())
+	if text != "" {
+		ioutil.WriteFile(tmpFile.Name(), []byte(text), 644)
+	}
+	return tmpFile.Name()
+}
+
+func cleanUpFiles() {
+	for _, file := range testFiles {
```

Comment:
would it be possible to put all the test files in a given tempdir, then clean up the whole dir?